### ISSUE TITLE
DiT Finetuning

### ISSUE THREAD

**Issue by zjq-Jackson** [jy0205/Pyramid-Flow#166]
"I only have one A800 80G; can I fine-tune the DiT model with it?"

**Comment by feifeiobama** [comment]
Sorry, you cannot do this with our current implementation. It is technically possible, but it would require a lot of code changes, e.g. accumulating the gradient for all sequence lengths in a video clip.

**Comment by zjq-Jackson** [comment]
Alright, thank you very much!

**Comment by lxxie298** [comment]
Thank you so much for sharing such meaningful work! Would it be feasible to fine-tune the DiT model on 256x256 low-resolution data using two A800 80G GPUs? @feifeiobama
